demo/cpp/model_peeker.cc

```diff
@@ -42,6 +43,13 @@ void peek_model(DLRModelHandle model) {
     std::cout << input_names[i] << ", ";
   }
   std::cout << std::endl;
+  output_names.resize(num_outputs);
+  std::cout << "output_names: ";
+  for (int i = 0; i < num_outputs; i++) {
+    GetDLROutputName(&model, i, &output_names[i]);
+    std::cout << output_names[i] << ", ";
+  }
+  std::cout << std::endl;
```

Comment:
Lets move it after `input_types:`, before `output shapes:`. Also, we should print `output_names` only if matadata is present.

Reply:
👍